public-images/public-images-reference/artifacts.rst

```diff
@@ -0,0 +1,1495 @@
+                                    "source_package_name": null,
+                                    "source_package_version": null,
+                                    "version": null
+                                },
+                                "cves": [],
+                                "launchpad_bugs_fixed": [],
+                                "changes": [],
+                                "notes": null
+                            },
+                            {
+                                "name": "linux-tools-6.8.0-35",
+                                "from_version": {
+                                    "source_package_name": "linux",
+                                    "source_package_version": "6.8.0-35.35",
+                                    "version": "6.8.0-35.35"
+                                },
+                                "to_version": {
+                                    "source_package_name": null,
+                                    "source_package_version": null,
+                                    "version": null
+                                },
+                                "cves": [],
+                                "launchpad_bugs_fixed": [],
+                                "changes": [],
+                                "notes": null
+                            },
+                            {
+                                "name": "linux-tools-6.8.0-35-generic",
+                                "from_version": {
+                                    "source_package_name": "linux",
+                                    "source_package_version": "6.8.0-35.35",
+                                    "version": "6.8.0-35.35"
+                                },
+                                "to_version": {
+                                    "source_package_name": null,
+                                    "source_package_version": null,
+                                    "version": null
+                                },
+                                "cves": [],
+                                "launchpad_bugs_fixed": [],
+                                "changes": [],
+                                "notes": null
+                            }
+                        ],
+                        "snap": []
+                    },
+                    "notes": "Changelog diff for Ubuntu 24.04 noble image from daily image serial 20240622 to 20240628",
+                    "from_series": "noble",
+                    "to_series": "noble",
+                    "from_serial": "20240622",
+                    "to_serial": "20240628",
+                    "from_manifest_filename": "daily_manifest.previous",
+                    "to_manifest_filename": "manifest.current"
+                }
+
+Checksums
+~~~~~~~~~
+.. list-table::
+   :widths: 1 2
+   :header-rows: 0
+
+   * - Extension
+     - ``<artifact>SUMS``
+   * - Example filename
+     - MD5SUMS, SHA256SUMS
```

Comment:
```suggestion
     - ``MD5SUMS``, ``SHA256SUMS``
```

Reply:
Thanks for catching these and the manifest, they were refactors near the end.